install/kubernetes/helm/istio/charts/kiali/templates/clusterrole.yaml

```diff
@@ -23,8 +23,6 @@ rules:
   - watch
 - apiGroups: ["config.istio.io"]
   resources:
-  - routerules
```

Comment:
I think we still need `routerules` for backward compatibility.

Reply:
kiali 0.5.0 is not accessing routerules directly anymore - it is using the new destinationrules. So it does not need routerules specified in its clusterrole